sdv/metadata/single_table.py

```diff
@@ -52,13 +52,16 @@ def _validate_numerical(self, column_name, **kwargs):
         representation = kwargs.get('representation')
         if representation and representation not in self._NUMERICAL_REPRESENTATIONS:
             raise ValueError(
-                f"Invalid value for 'representation' {representation} for column '{column_name}'.")
+                f"Invalid value for 'representation' '{representation}'"
+                f" for column '{column_name}'."
+            )
 
     @staticmethod
     def _validate_datetime(column_name, **kwargs):
         datetime_format = kwargs.get('datetime_format')
-        if datetime_format:
+        if datetime_format is not None:
             try:
+                # NOTE: I don't know if this ever crashes, it just returns the string as is
```

Comment:
@pvk-developer I'm not sure how `strftime` works, but it doesn't seem to raise errors even if you pass it a normal string like 'value'